web-admin/tests/bookmarks.spec.ts

```diff
@@ -83,20 +90,25 @@ test.describe("Bookmarks", () => {
 
       // Remove all filters. Home Bookmarks shouldn't apply.
       // This is as per requirement where clearing filters should take the user to the default set of values.
-      await page.goto("/e2e/openrtb/explore/auction_explore_bookmarks");
-      await expect(page.getByText("Last 7 Days")).toBeVisible();
-      await expect(page.getByText("No filters selected")).toBeVisible();
-      await expect(page.getByText("no comparison period")).toBeVisible();
-      await expect(
-        page.getByLabel("Leaderboards", { exact: true }),
-      ).toBeVisible();
+      // TODO: revisit if this is correct
+      // await page.goto("/e2e/openrtb/explore/auction_explore_bookmarks");
+      // await expect(page.getByText("Last 7 Days")).toBeVisible();
+      // await expect(page.getByText("No filters selected")).toBeVisible();
+      // await expect(page.getByText("no comparison period")).toBeVisible();
+      // await expect(
+      //   page.getByLabel("Leaderboards", { exact: true }),
+      // ).toBeVisible();
```

Comment:
What's going on here? If it were uncommented, I don't see any user actions that "remove all filters"?

Reply:
It was a test for https://github.com/rilldata/rill/issues/6248

But both Mike and Alex were confused by this behaviour. So we are getting rid of it for now.